/tmp/ccp1dL7W.o: En la funci√≥n `themainmainfunction':
shield_c.c:(.text+0x15): referencia a `errorNo1' sin definir
collect2: error: ld returned 1 exit status
